tapiro/ci/test.sh

```diff
@@ -0,0 +1,7 @@
+#! /bin/sh
+
+set -e
+
+apk add --no-cache curl
+
+sbt -batch ';tapiroCore/compile ;sbt-tapiro/scripted' # TODO(claudio): compile to test once we have them
```

Comment:
I think you either need to add a `cd retro` before this, or use the `dir` option in `run` in the yaml task

Reply:
Ah, you were faster than my review :)

Reply:
👍 looks like it's running now, so the permission should be fine (it was created with permission 755 I think)